.github/workflows/ci.yml

```diff
@@ -14,10 +14,13 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        python-version: ['3.8']
+        python-version: ['3.8', '3.12']
         toxenv: [
-            quality, docs, django32-drf314, django42-drf314
+            quality, docs, django42-drf314
         ]
+        exclude:
+          - python-version: '3.12'
```

Comment:
quality checks for `python 3.12` require some fixes which don't work in `quality-pyton3.8` so we can't run both at the same time. 
This exclusion can be removed once the 3.8 support is completely removed after upgrading dependencies with python 3.12.